src/bluesearch/mining/eval.py

```diff
@@ -78,8 +78,7 @@ def _check_consistent_iob(iob_true: pd.Series, iob_pred: pd.Series) -> None:
             )
 
         etypes = unique_etypes(x)
-        # cast() needed to tell mypy that fillna() doesn't return None here.
-        x_prev = cast(pd.Series, x.shift(periods=1).fillna("O", inplace=False))
+        x_prev = x.shift(periods=1).fillna("O", inplace=False)
```

Comment:
It is nice that we don't need to use `cast` anymore! It is thanks to the upgrade of the `pandas_stubs` ?

Reply:
Yes I think so, once I upgraded `pandas_stubs` it started saying the cast was unnecessary.